src/main/java/ast/psi/PsiParameterListDelegate.java

```diff
@@ -12,4 +13,12 @@
 	protected PsiParameterListDelegate(NodeConfig<? extends PsiParameterList> config) {
 		super(config);
 	}
+
+	public void getParameterList() {
+		PsiParameterList list = (PsiParameterList) this.element;
+		PsiParameter[] params = list.getParameters();
+		for (PsiParameter p: params) {
+			this.delegateStore.getNodeFrom(p);
+		}
+	}
```

Comment:
This is how I getParameterList from the `PsiParameterList`, but I am not sure how this relates to the `PsiCallExpressionDelegate` class. Could you point it out?